### PR TITLE
updated findings from Nika

### PR DESCRIPTION
**Pull request by abdallahshaban557**
_Issue #, if available:_

Updates to docs based on feedback from Nika

Also fixes https://github.com/aws-amplify/docs/issues/4837

_Description of changes:_

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/fragments/lib/project-setup/flutter/create-application/30_provisionBackend.mdx [hunk 1]
```diff
@@ -44,8 +44,8 @@ After the project information is filled, you will be prompted to select the AWS
 Please choose the profile you want to use (Use arrow keys)
 > default
 ```
-> Where possible the CLI will infer the proper configuration based on the type of project Amplify is being initialized in. In this case it knew you are using Create React App and provided the proper configuration for type of app, framework, source, distribution, build, and start options.
 
+> Where possible the CLI will infer the proper configuration based on the type of project Amplify is being initialized in. In this case it knew you are using a Flutter app, source, distribution, build, and start options.
```
Comment (tannerabread) [inline src/fragments/lib/project-setup/flutter/create-application/30_provisionBackend.mdx:48]:
```suggestion
> Where possible, the CLI will infer the proper configuration based on the type of project Amplify is being initialized in. In this case, it knew you are using a Flutter app, source, distribution, build, and start options.
```

Comment (tannerabread) [reply src/fragments/lib/project-setup/flutter/create-application/30_provisionBackend.mdx:48]: not a blocking change but I think commas would go in these 2 spots?

**Review by tannerabread — DISMISSED** [review]
one non-blocking suggestion but otherwise LGTM!

**Review by tannerabread — APPROVED** [review]
:shipit:

**Review by tannerabread — APPROVED** [review]
LGTM

**Review by katiegoines — APPROVED** [review]
LGTM!

**Pull request merged**